test_install/CMakeLists.txt

```diff
@@ -2,7 +2,9 @@ cmake_minimum_required(VERSION 3.8)
 
 project(TestInstall LANGUAGES CXX)
 
-# `enable_language(CUDA)` is needed when ginkgo was statically build with CUDA
+if (NOT GINKGO_BUILD_SHARED_LIBS)
+    enable_language(CUDA)
+endif()
```

Comment:
I think we no longer need this because some of your CMake modernizations made this step unnecessary.

Reply:
Right, thanks for catching this I have forgotten to update this MR to remove that part.